scripts/projects.py

```diff
@@ -130,7 +131,81 @@ def fuzz(self, jobs=CORES, fuzztime=3600, dump=False):
             os.system(f"mv {OSSFUZZ}/build/out/{self.project}/crash-* {crash_dir}")
 
     def postprocess(self):
-        pass
+        proj_name = self.project
+        json_path = os.path.join(OSSFUZZ_SCRIPTS_HOME, "dump", proj_name)
+        json_file_names = [f for f in os.listdir(json_path) if f.endswith(".json")]
+        info(f"Looking for json file: {json_file_names}")
+
+        for json_file_name in json_file_names:
+            file_name = json_file_name.split(".json")[0]
+            # open Json file and get filename+func_name
+            with open(os.path.join(json_path, json_file_name), "r") as f:
+                try:
+                    data = json.load(f)
+                except json.JSONDecodeError:
+                    warning(f" {json_file_name}  Json file illegal")
+                    continue
+            info(f"NUMBER OF FUNCTION: {len(data)}")
+            pre_file_path, code_content = "", ""
+            for cnt in range(len(data)):
+                file_func_name = ""
+                for file_func_name_ in data[cnt]:
+                    file_func_name = file_func_name_
+
+                # if split not matched, need to add a check
+                splited_file_func_name = file_func_name.split("?")
```

Comment:
can you put something like `self.file_func_delim = "?"` in `__init__`  and use it here? It might be easier for later works if we want to change anything.